humblegen-rt/src/server.rs

```diff
@@ -52,8 +54,14 @@ pub async fn handle_request(
     services: Arc<RegexSetMap<Request<Body>, Service>>,
     req: Request<Body>,
 ) -> Response<Body> {
+    let request_id: String = rand::thread_rng()
+        .sample_iter(&rand::distributions::Alphanumeric)
+        .take(10)
```

Comment:
This is ~ 60 bits of entropy, correct? A uuid typically a little under 128, maybe we should go up to at least 20  chars here?

Reply:
According to http://davidjohnstone.net/pages/hash-collision-probability, with 60 bits and 10 million requests, we're already at 0.004% collision chance, which seems a bit too close for comfort... =)

Reply:
Have you thought about the goal we are trying to achieve here? Attributing log entries, not having an almost-guaranteed collision-freeness.